public/css/docs-highlight.css

```diff
@@ -52,8 +52,8 @@
   --hl-namespace: var(--dk-pink);
 
   --hl-highlight-line: #202641;
-  --hl-deletion: #461e1f;
-  --hl-addition: #14301d;
+  --hl-deletion: #ec5f67;
+  --hl-addition: #99c794;
```

Comment:
Where'd these come from? 🧑🏻‍🎨

Reply:
From begin.com hljs diff colors. If you look at a diff in the docs you will not be able to read it.